### PR TITLE
Avoid environment in inits

### PR DESCRIPTION
**Pull request by pconesa**
This will just avoid using the environment at import time. for 2 variables.

### CODE REVIEW

**Comment by pconesa** [comment]
wait, this started from fixes, review after fixes

**Comment by pconesa** [comment]
this is breaking python3 installation. I'm looking at it.

File: pyworkflow/tests/model/tests_logs.py [hunk 1]
```diff
@@ -4,8 +4,9 @@
 import logging
 import unittest
 
+from pyworkflow import Config
 from pyworkflow.utils.utils import getLineInFile, isInFile
-from pyworkflow.utils.log import ScipionLogger, LOG_FILE
+from pyworkflow.utils.log import ScipionLogger, getLogConfiguration
```
Comment (delarosatrevin) [inline pyworkflow/tests/model/tests_logs.py]: @pconesa we should keep pyworkflow.utils as the submodule to import...all other files underneath should be implementation of functionality. Could you check if 'log' is imported in the init? Previous line to this one should also be changed.

File: pyworkflow/utils/log.py [hunk 2]
```diff
@@ -26,54 +26,53 @@
...
 import logging
 import logging.config
-from pyworkflow.utils.path import makeFilePath
-
-
 
-# Get general log file path
-LOG_FILE = os.path.join(os.environ['SCIPION_LOGS'], 'scipion.log')
+from pyworkflow import Config
+from pyworkflow.utils.path import makeFilePath
```
Comment (delarosatrevin) [inline pyworkflow/utils/log.py]: Same here...'path' should not be directly used for import

File: pyworkflow/tests/model/tests_logs.py (hunk 1 above)
Comment (pconesa) [reply pyworkflow/tests/model/tests_logs.py]: Do you mean to add this "from log import *" at the utils.__init__?

Comment (delarosatrevin) [reply pyworkflow/tests/model/tests_logs.py]: Yep. And then from outside utils module every import should be at the level of pyworkflow.utils

File: pyworkflow/utils/log.py (hunk 2 above)
Comment (pconesa) [reply pyworkflow/utils/log.py]: Should we do then:
import pyworkflow.utils as pwutils

and then ...pwutils.whatevertoreachthecode?

Comment (delarosatrevin) [reply pyworkflow/utils/log.py]: Yes, I personally prefer this style...but this is how you want to make the import. The other comment is more what is hide/exposed by the module, i.e, never import something from an internal file of the 'submodule', unless it is a well-defined sub-submodule...I think we have improved that...but still a few places with this kind of imports remaining.

Comment (pconesa) [reply pyworkflow/utils/log.py]: ok, doing it thanks

**Comment by pconesa** [comment]
Done.

**Comment by pconesa** [comment]
thanks to you! @delarosatrevin for reviewing it.

**Pull request merged**